src/pal/src/exception/seh-unwind.cpp

```diff
@@ -223,6 +223,11 @@ RaiseException(IN DWORD dwExceptionCode,
         dwExceptionCode ^= RESERVED_SEH_BIT;
     }
 
+    if (nNumberOfArguments > EXCEPTION_MAXIMUM_PARAMETERS) 
```

Comment:
I've missed the check below, mostly because it was incorrectly setting the exceptionRecord.NumberParameters to the larger value and one of the exception tests was failing due to that. So I'll preserve the check where I've put it, add the warning and delete the duplicate one.